### ISSUE TITLE
Container fails to start

### ISSUE THREAD

**Issue by aniajj9** [Azure-Samples/ctfd-azure-paas#13]
> Please provide us with the following information:
> ---------------------------------------------------------------

### This issue is for a: (mark with an `x`)
```
- [x] bug report -> please search issues before submitting
- [ ] feature request
- [ ] documentation issue or request
- [ ] regression (a behavior that used to work and stopped in a new release)
```

### Minimal steps to reproduce
> Deploy to azure using the "Deploy to Azure" button

### Any log messages given by the failure
> 2023-07-14T10:23:25.987Z ERROR - Container ctfd-app-jiibwegyv3xy6_1_240b2df1 for site ctfd-app-jiibwegyv3xy6 did not start within expected time limit. Elapsed time = 230.6916131 sec
2023-07-14T10:23:26.031Z ERROR - Container ctfd-app-jiibwegyv3xy6_1_240b2df1 didn't respond to HTTP pings on port: 8000, failing site start. See container logs for debugging.
2023-07-14T10:23:26.076Z INFO  - Stopping site ctfd-app-jiibwegyv3xy6 because it failed during startup.

### Expected/desired behavior
> 504.0 GatewayTimeout on the website that I am trying to open

### OS and Version?
> Windows 11

### Versions
> latest

### Mention any other details that might be useful
> The deployment seems to be successful, without any errors
> It seems to be the same error as the one previously raised in a closed thread with the same name, however redeploying did not seem to solve the issue, as it did in the other person's case :( 

> ---------------------------------------------------------------
> Thanks! We'll be in touch soon.
[logs.zip](https://github.com/Azure-Samples/ctfd-azure-paas/files/12049519/logs.zip)

**Comment by balteravishay** [comment]
Thanks for reporting this @aniajj9 , i'll fix the issue and hope to get a patch out soon.

**Comment by balteravishay** [comment]
fixed by #16
